google/cloud/spanner/keys_test.cc

```diff
@@ -48,6 +48,46 @@ TEST(KeySetTest, AllKeys) {
   EXPECT_THAT(result, spanner_testing::IsProtoEqual(expected));
 }
 
+TEST(KeySetTest, EqualityEmpty) {
+  KeySet expected;
+  KeySet actual;
+  EXPECT_TRUE(expected == actual);
```

Comment:
I think it's good to prefer `EXPECT_EQ` and `EXPECT_NE`, because they have the benefit of strinifying the arguments when the expectations fail. We wouldn't get that with a failed `EXPECT_TRUE`

Reply:
Done.